.github/workflows/centos.yml

```diff
@@ -0,0 +1,237 @@
+name: CentOS
+
+on:
+  push:
+    branches:
+      - main
+      - feature/*
+      - merge*
+      - fix/*
+  release:
+    types: [ created ]
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+
+  build_sirius:
+    name: build sirius
+    runs-on: ubuntu-latest
+    container: 'centos:centos7'
+    env:
+      Sirius_Ref: antares_integration
+    steps:
+      - name: Get release
+        if: github.event_name == 'release' && github.event.action == 'created'
```

Comment:
Since `github.event_name == 'release' && github.event.action == 'created'` appears 6 times, it would make sense to store it in a new variable.